bin/sidecli.re

```diff
@@ -15,13 +15,38 @@ let read_file = file => {
     );
   await(lines |> String.concat("\n"));
 };
-let read_file = file => read_file(file) |> Lwt_main.run;
+
 let read_identity_file = file => {
-  let file_buffer = read_file(file);
+  let file_buffer = read_file(file) |> Lwt_main.run;
   let json = Yojson.Safe.from_string(file_buffer);
   identity_of_yojson(json) |> Result.get_ok;
 };
 
+let write_file = (~file, string) => {
+  let oc = open_out(file);
+  output_string(oc, string);
+  close_out(oc);
+};
+
+let read_validators = file => {
+  let.await file_buffer = read_file(file);
+  await(
+    try({
+      let json = Yojson.Safe.from_string(file_buffer);
+      module T = {
+        [@deriving of_yojson]
+        type t = {
```

Comment:
Not blocking, because it's not any worse than it was before, but this is duplicated [here](https://github.com/marigold-dev/sidechain/pull/44/files#diff-f199cc2f35dec0a8571164460a4a4bfa6ca3100739f3f5b69314008de807ccfcR379) and we should remember to merge them

Reply:
@anchpop open an issue

Reply:
resolving the conversation to merge, but with a follow up #60